scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -49,17 +51,21 @@ def __init__(self,
                 test. This is the main case associated with the test.
             separate_builds (bool): Whether separate builds are needed for the
                 two cases. If False, case2 uses the case1 executable.
+            separate_builds (bool): Whether separate runs are needed for the
+                two cases. If False, case2 uses the case1 run area. Can be useful for restart tests
             run_two_suffix (str, optional): Suffix appended to the case name for
                 the second run. Defaults to 'test'. This can be anything other
                 than 'base'.
             run_one_description (str, optional): Description printed to log file
                 when starting the first run. Defaults to ''.
             run_two_description (str, optional): Description printed to log file
                 when starting the second run. Defaults to ''.
+            multisubmit (bool): Do first and second runs as different submissions
```

Comment:
Also, I'm a bit suspicious of whether the ERR and ERRI tests really work correctly with this new infrastructure. It looks to me like RESUBMIT will be set to 1 in both cases, since this is set via config_tests.xml. Does that do the right thing? If so, it's not very intuitive, and should be documented somewhere.

It also seems like ERR and ERRI are not re-runnable, because presumably RESUBMIT gets decremented to 0, and then if you try to rerun them they'd do the wrong thing because RESUBMIT would be 0 rather than 1. But if that's truly an issue, I don't think it's new to this PR.